### PR TITLE
test(e2e): run all tests against tab navigator

### PR DESCRIPTION
**Pull request by satish-ravi**
### Description

As part of removing the drawer nav, run all e2e tests against the tab navigator and remove drawer nav specific tests. WalletConenct test still runs against drawer because it uses openUrl to launch app and there isn't an easy way to specify a launchArg

### Test plan

CI

### Related issues

- Part of ACT-1133

### Backwards compatibility

Yes

### Network scalability

Yes

### CODE REVIEW

File: e2e/src/HomeFeed.spec.js [hunk 1]
```diff
@@ -7,49 +7,47 @@ beforeAll(async () => {
...
-  ':ios: Home Feed - $navType',
-  ({ navType }) => {
-    beforeAll(async () => {
-      await launchApp({
-        newInstance: true,
-        permissions: { notifications: 'YES', contacts: 'YES', camera: 'YES' },
-        launchArgs: { statsigGateOverrides: `use_tab_navigator=${navType === 'tabs'}` },
-      })
+describe(':ios: Home Feed', () => {
+  // TODO(ACT-1133): remove this beforeAll when tab tests are removed
```
Comment (finnian0826) [inline e2e/src/HomeFeed.spec.js:11]: What tab tests are being removed?

File: e2e/src/usecases/ResetAccount.js [hunk 2]
```diff
@@ -2,14 +2,14 @@ import { SAMPLE_BACKUP_KEY } from '../utils/consts'
...
-export default ResetAccount = ({ navType }) => {
+export default ResetAccount = () => {
   // TODO(ACT-1133): remove this launchApp as it is only needed to update
   // statsig gate overrides
   beforeAll(async () => {
     await launchApp({
       newInstance: true,
       permissions: { notifications: 'YES', contacts: 'YES' },
-      launchArgs: { statsigGateOverrides: `use_tab_navigator=${navType === 'tab'}` },
+      launchArgs: { statsigGateOverrides: `use_tab_navigator=tab` },
```
Comment (finnian0826) [inline e2e/src/usecases/ResetAccount.js:12]: Add todo for removing launchArgs

File: e2e/src/usecases/Settings.js [hunk 3]
```diff
@@ -3,20 +3,20 @@ import { reloadReactNative, launchApp } from '../utils/retries'
...
-export default Settings = ({ navType }) => {
+export default Settings = () => {
   // TODO(ACT-1133): remove this launchApp as it is only needed to update
   // statsig gate overrides
   beforeAll(async () => {
     await launchApp({
       newInstance: true,
       permissions: { notifications: 'YES', contacts: 'YES' },
-      launchArgs: { statsigGateOverrides: `use_tab_navigator=${navType === 'tab'}` },
+      launchArgs: { statsigGateOverrides: `use_tab_navigator=true` },
```
Comment (finnian0826) [inline e2e/src/usecases/Settings.js:13]: Add todo for removing launchArgs

File: e2e/src/HomeFeed.spec.js (hunk 1 above)
Comment (satish-ravi) [reply e2e/src/HomeFeed.spec.js:11]: was a typo, fixed

File: e2e/src/usecases/ResetAccount.js (hunk 2 above)
Comment (satish-ravi) [reply e2e/src/usecases/ResetAccount.js:12]: see todo 5 lines above

File: e2e/src/usecases/Settings.js (hunk 3 above)
Comment (satish-ravi) [reply e2e/src/usecases/Settings.js:13]: see todo 5 lines above

**Review by MuckT — APPROVED** [review]

**Pull request merged**